recipe_recommender/bodybuilding.py

```diff
@@ -94,3 +101,95 @@ def save_df():
 
     # Save as csv
     df.to_csv(save_path)
+
+
+def process_ingredients(data):
+    """
+       Process the ingredients from bodybuilding.com and save the recipe
+
+       data is the scraped json.
+
+    """
+
+    all_ingredients = []
+    all_amounts = []
+    all_units = []
+    all_descriptors = []
+    all_units = []
+    recipe_ids = []
+
+    for dat in data:
+        recipe_id = dat['id']
+        dat = dat['schemaOrg']
+        if 'recipeIngredient' in dat.keys():
+            ingredients = dat['recipeIngredient']
+            for ingredient in ingredients:
+                descriptor = None
+                recipe_ids.append(recipe_id)
+
```

Comment:
Can you add a comment here to explain what's being done? I feel like I only understand because we talked about it.